enclave-manager/web/package.json

```diff
@@ -31,8 +31,6 @@
     "analyze": "lerna run analyze",
     "cleanInstall": "rm -rf node_modules; rm -rf packages/*/node_modules; yarn install; lerna init",
     "clean": "lerna run clean",
-    "prebuild": "rm -rf ../../engine/server/webapp",
-    "postbuild": "cp -r packages/app/build/ ../../engine/server/webapp",
```

Comment:
@Dartoxian won't you still need this if you want to build a development engine and push it to a cloud instance (like from your laptop)?

Reply:
Also another thing to think about, if you run the main (root) build script will that include the EM UI? It needs to, especially if you want to test locally before pushing and eventually making a PR.

Reply:
yep thanks for catching this @adschwartz - I've updated the pr and tested `scripts/build.sh` is producing a working `kurtosis web` locally (and validated that it wasn't prior to this update).